SpotifyAPI.Web/Models/Response/QueueResponse.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+
+namespace SpotifyAPI.Web
+{
+  public class QueueResponse
+  {
+    public FullTrack CurrentlyPlaying { get; set; } = default!;
```

Comment:
I think we should change this to:

```c#
    [JsonConverter(typeof(PlayableItemConverter))]
    public IPlayableItem CurrentlyPlaying { get; set; } = default!;
    [JsonProperty(ItemConverterType = typeof(PlayableItemConverter))]
    public List<IPlayableItem> Queue { get; set; } = default!;
```

While the documentation wrongly says only `type=track` is returned, the API happily also returns `Episodes` in the `CurrentlyPlaying` and `Queue` fields